### PR TITLE
Add(doc): custom operator image

### PR DESCRIPTION
**Pull request by clamoriniere**
### What does this PR do?

Add instruction to create a custom container image with a CA certificat

### Motivation

document user support case request

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Write there any instructions and details you may have to test your PR.

### CODE REVIEW

File: docs/custom-operator-image.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Customer Operator container image
+
+In some case, building a customer Operator image is needed, for example: to add a customer secret backend implementation, or to add CA certificat.
```
Comment (rtrieu) [inline docs/custom-operator-image.md:3]: Can you provide some more context about what a custom operator image is and explain how it's used in Datadog? This sentence jumps right into "In some cases..." but it's not clear what a user might be doing at the time they'll need to build a customer operator image.

File: docs/installation.md [hunk 2]
```diff
@@ -185,6 +185,18 @@ datadog-agent-zvdbw                          1/1     Running    0          8m1s
 
 See the [`kubectl` plugin doc](/docs/kubectl-plugin.md)
 
+
+## Use a custom Datadog Operator image
+
+you can find the instruction to build a Datadog Operator custom container image based on an official release [custom-operator-image.md][9]
+
+### with the helm chart
```
Comment (rtrieu) [inline docs/installation.md:193]:
```suggestion
### Datadog Operator images with Helm charts

To use a Datadog operator image with a Helm chart:

```

File: docs/installation.md [hunk 3]
```diff
@@ -185,6 +185,18 @@ datadog-agent-zvdbw                          1/1     Running    0          8m1s
 
 See the [`kubectl` plugin doc](/docs/kubectl-plugin.md)
 
+
+## Use a custom Datadog Operator image
+
+you can find the instruction to build a Datadog Operator custom container image based on an official release [custom-operator-image.md][9]
```
Comment (rtrieu) [inline docs/installation.md:191]:
```suggestion
See instructions to build a Datadog Operator custom container image on an official release in [Custom Operator container images][9].
```

File: docs/custom-operator-image.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Customer Operator container image
```
Comment (rtrieu) [inline docs/custom-operator-image.md:1]:
```suggestion
# Customer Operator container images
```

File: docs/custom-operator-image.md (hunk 1 above)
Comment (rtrieu) [inline docs/custom-operator-image.md:3]:
```suggestion
When adding a customer secret backend implementation or adding a CA certificate, for example, you may need to build a customer Operator image.
```

File: docs/custom-operator-image.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+In some case, building a customer Operator image is needed, for example: to add a customer secret backend implementation, or to add CA certificat.
+
+## Add a CA certificat
```
Comment (rtrieu) [inline docs/custom-operator-image.md:5]:
```suggestion
## Add a Certificate Authority (CA) certificate
```

File: docs/custom-operator-image.md (hunk 4 above)
Comment (rtrieu) [reply docs/custom-operator-image.md:1]: Just double checking - do you mean "Customer Operator container images" here or "Custom Operator container images"?

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some changes requested before approving!

File: docs/custom-operator-image.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Customer Operator container images
```
Comment (celenechang) [inline docs/custom-operator-image.md:1]:
```suggestion
# Custom Operator container images
```

File: docs/custom-operator-image.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+When adding a customer secret backend implementation or adding a CA certificate, for example, you may need to build a customer Operator image.
```
Comment (celenechang) [inline docs/custom-operator-image.md:3]:
```suggestion
When adding a custom secret backend implementation or adding a CA certificate, for example, you may need to build a custom Operator image.
```

File: docs/installation.md [hunk 8]
```diff
@@ -185,6 +185,21 @@ datadog-agent-zvdbw                          1/1     Running    0          8m1s
 
 See the [`kubectl` plugin doc](/docs/kubectl-plugin.md)
 
+
+## Use a custom Datadog Operator image
+
+See instructions to build a Datadog Operator custom container image on an official release in [Custom Operator container images][9].
```
Comment (celenechang) [inline docs/installation.md:191]:
```suggestion
See instructions to build a Datadog Operator custom container image based on an official release in [Custom Operator container images][9].
```

File: docs/installation.md [hunk 9]
```diff
@@ -185,6 +185,21 @@ datadog-agent-zvdbw                          1/1     Running    0          8m1s
...
+
+### Datadog Operator images with Helm charts
+
+To use a Datadog operator image with a Helm chart:
```
Comment (celenechang) [inline docs/installation.md:195]:
```suggestion
To install a custom Datadog Operator image using the Helm chart, run the following:
```

File: docs/custom-operator-image.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+
+```dockerfile
+FROM gcr.io/datadoghq/operator:latest
+
+COPY my-ca-certificat.crt /usr/share/pki/ca-trust-source/anchors/
+
+## update-ca-trust needs to run as root
+USER root
+RUN update-ca-trust
+## Run trust list to check that the certificat was properly added
```
Comment (celenechang) [inline docs/custom-operator-image.md:17]:
```suggestion
## Run trust list to check that the certificate was properly added
```

**Review by celenechang — APPROVED** [review]

**Pull request merged**